test/e2e/kuma_deploy_hybrid_test.go

```diff
@@ -234,4 +243,27 @@ metadata:
 		Expect(err).ToNot(HaveOccurred())
 		Expect(stdout).To(ContainSubstring("HTTP/1.1 200 OK"))
 	})
+
+	It("should sync traffic permissions", func() {
+		err := global.GetKumactlOptions().KumactlDelete("traffic-permission", "traffic-permission-all")
+		Expect(err).ToNot(HaveOccurred())
+
+		err = YamlUniversal(trafficPermissionToKuma2)(global)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Remote 3
```

Comment:
Is there `Remote 3`? I thought there was only Remote 1 and Remote 2

Reply:
In Hybrid - 3 and 4 are the Docker/Universal clusters